packages/dashboard-backend/LICENSE

```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```

Comment:
Do we need LICENSE in each of package?

Reply:
I suppose we don't need it, but let's do the cleanup after this PR is merged.